contribs/gmf/examples/style/mobileapp-search.less

```diff
@@ -0,0 +1,44 @@
+.tt-menu {
+  position: fixed!important;
+  left: 0;
+  right: 0;
+  top: 50px!important;
+  bottom: 0;
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 100vw;
+  background-color: white;
+  border: solid 1px black;
+  border-top: none;
+  .search-header {
+    display: none;
+  }
+  .search-datum {
+    border-top: solid 1px black;
+    margin: 0;
+    padding: 10px @app-margin;
+    text-align: left;
+    &:hover {
+      cursor: pointer;
+      background-color: @onhover-color;
+    }
+    p {
+      margin: 0;
+    }
+    .search-label {
+      color: @search-result-font-color;
+    }
+    .search-group {
+      color: lighten(@search-result-font-color, 50%);
+      font-size: 0.8em;
+    }
+  }
+}
+
+//For tablet only
+@media (min-width: @screen-sm-min) {
+  .tt-menu {
+    width: 33vw;
+    border-top-right-radius: 20px;
```

Comment:
Is there any reason for this radius on the top right border? Cosmetic ?
Personaly I don't like it.  ;-)

![screenshot from 2015-11-17 16 06 13](https://cloud.githubusercontent.com/assets/319774/11215163/2ad2a53a-8d45-11e5-8980-eb6a390a8c5e.png)